packages/compass-import-export/src/components/export-field-options.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import {
+  Banner,
+  Label,
+  RadioBox,
+  RadioBoxGroup,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+import type { FieldsToExportOption } from '../modules/export';
+
+const selectFieldsRadioBoxStyles = css({
+  // Keep the label from going to two lines.
+  whiteSpace: 'nowrap',
+});
+
+const messageBannerStyles = css({
+  marginTop: spacing[3],
+});
+
+const selectFieldsToExportId = 'select-fields-to-export';
+const selectFieldsToExportLabelId = 'select-fields-to-export-label';
+
+function FieldsToExportOptions({
+  fieldsToExportOption,
+  setFieldsToExportOption,
+}: {
+  fieldsToExportOption: FieldsToExportOption;
+  setFieldsToExportOption: (fieldsToExportOption: FieldsToExportOption) => void;
+}) {
+  const [showProjectInfoMessage, setShowProjectInfoMessage] =
+    useState<boolean>(true);
+
+  return (
+    <>
+      <Label htmlFor={selectFieldsToExportId} id={selectFieldsToExportLabelId}>
+        Fields to export
+      </Label>
+      <RadioBoxGroup
+        aria-labelledby={selectFieldsToExportLabelId}
+        data-testid="select-file-type"
+        id={selectFieldsToExportId}
+        onChange={({
+          target: { value },
+        }: React.ChangeEvent<HTMLInputElement>) =>
+          setFieldsToExportOption(value as FieldsToExportOption)
+        }
+      >
+        <RadioBox
+          data-testid="select-file-type-json"
+          value="all-fields"
+          checked={fieldsToExportOption === 'all-fields'}
+        >
+          All fields
+        </RadioBox>
+        <RadioBox
+          className={selectFieldsRadioBoxStyles}
+          data-testid="select-file-type-csv"
+          value="select-fields"
+          checked={fieldsToExportOption === 'select-fields'}
+        >
+          Select fields in table
+        </RadioBox>
+      </RadioBoxGroup>
+      {showProjectInfoMessage && (
+        <Banner
+          className={messageBannerStyles}
+          dismissible
+          onClose={() => setShowProjectInfoMessage(false)}
```

Comment:
Why do we bother making some of these dismissable? 😄

Reply:
Those were in the designs and I think I the pattern is information about what you can do vs information on what's going to happen. Things you can do with some additional info are dismissible, while banners with information about what will happen are not dismissible.

Reply:
I just see it as added complexity for zero gain. It doesn't even remember that you closed it so it will just reappear next time. But it is here now, so might as well keep it.